spec/lib/gds_api/whitehall_export_spec.rb

```diff
@@ -8,16 +8,16 @@
     let(:whitehall_export_page_2) { build(:whitehall_export_index, documents: build_list(:whitehall_export_index_document, 10)) }
 
     before do
-      stub_request(:get, "#{whitehall_host}/government/admin/export/document?lead_organisation=123&type=NewsArticle&page_count=100&page_number=1")
+      stub_request(:get, "#{whitehall_host}/government/admin/export/document?lead_organisation=123&type=NewsArticle&subtypes[]=NewsStory&subtypes[]=PressRelease&page_count=100&page_number=1")
```

Comment:
Since these API requests are quite lengthy, we could extract it out to a helper method.  We have an example of this already [here](https://github.com/alphagov/content-publisher/blob/master/spec/support/topics_helper.rb#L6).  It might be premature refactoring to create a new helper file at this stage, but we could still have the method in this file.  We'd also have the added bonus of being able to do something like this when we're asserting that the request has been made:
```
api_request = stub_whitehall_export_api_document
expect(whitehall_document_list.next).to have_requested(api_request)
```

Reply:
I've moved all the stubbing into a separate method and am now asserting against the stub, rather than having a URL hardcoded twice.